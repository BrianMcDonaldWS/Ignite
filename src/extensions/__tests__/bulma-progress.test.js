import MarkdownIt from 'markdown-it';
import progress from '../bulma-progress';

const md = new MarkdownIt().use(progress);

test('bulma - progress', () => {
  expect(
    md.render("'%% 75 is-primary is-large Look at all this progress! %%")
  ).toMatchSnapshot();
});
