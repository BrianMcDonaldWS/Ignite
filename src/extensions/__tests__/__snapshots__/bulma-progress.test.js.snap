// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bulma - progress 1`] = `
"<p>'<progress class=\\"progress\\" value=\\"22\\" max=\\"100\\">22%</progress></p>
"
`;

exports[`bulma - progress 2`] = `
"<p>'
    <div class=\\"progress-with-message\\">
      <progress class=\\"progress is-primary is-large\\" value=\\"75\\" max=\\"100\\">
        75%
      </progress>
      <span class=\\"progress-message\\">Look at all this progress!</span>
    </div>
  </p>
"
`;
